src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -36,7 +36,7 @@
     public static final Block blockHutBarracks            = new BlockHutBarracks();
     public static final Block blockHutBarracksTower       = new BlockHutBarracksTower();
     public static final Block blockInfoPoster             = new BlockInfoPoster();
-
+    public static final Block blockHutCook             = new BlockHutCook();
 
     // Deactivated for now
     // public static final Block blockBarrel        = new BlockBarrel();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)